addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -81,79 +81,35 @@
 </template>
 
 <!-- Extend default mass_mailing snippets with website feature -->
-<template id="social_links">
+<template id="mass_mailing.social_links">
```

Comment:
Should not replace the original template but extend it with xpaths.

Reply:
Done changes...!